styles.css

```diff
@@ -99,18 +100,303 @@ html {
 }
 
 .illustration {
-  background-image: url("/images/illustration.svg");
+  background-image: url("./images/illustration.svg");
   background-repeat: no-repeat;
   background-position: -20px 230px;
   width: 100%;
   height: 100%;
-  z-index: 1;
   display: inline-block;
 }
 
+.works-title {
+  font-family: "Crete Round", serif;
+  font-weight: 400;
+  color: #1c1a19;
+  font-size: 40px;
+  text-align: center;
+  margin-top: 100px;
+}
+
+.works-title-line {
+  width: 90%;
+  height: 2px;
+  background-color: #c7c7c7;
+  margin: 50px auto;
+}
+
+.works-wrapper {
+  width: 100%;
+  display: grid;
+  gap: 2rem;
+  grid-template-columns: 1fr;
+}
+
+.work-project-wrapper {
+  display: flex;
+  flex-direction: column;
+  width: 32.5rem;
+  margin: 0 auto;
+}
+
+.work-project-img {
+  width: 32.5rem;
+  margin: 0 auto;
+}
+
+.work-project-title {
+  font-family: var(--font-crete-round);
+  color: var(--color-primary);
+  font-weight: 400;
+  font-size: 3.2rem;
+  margin: 4rem 0;
+}
+
+.work-project-description {
+  font-family: var(--font-inter);
+  color: var(--color-primary);
+  font-weight: 400;
+  font-size: 1.6rem;
+  line-height: 2.4rem;
+}
+
+.work-info-tags {
+  display: flex;
+  list-style-type: none;
+  margin: 1rem 0;
+}
+
+.work-info-tag {
+  font-family: var(--font-inter);
+  color: var(--color-primary);
+  border: 0.1em solid #8993a4;
+  margin: 1rem 0;
+  margin-right: 1rem;
+  padding: 1rem;
+  font-size: 1.5rem;
+}
+
+.see-work-btn {
+  background-color: var(--color-secondary);
+  color: var(--color-white);
+  font-family: var(--font-inter);
+  font-size: 1.7rem;
+  font-weight: 600;
+  text-decoration: none;
+  padding: 1rem 2rem;
+  margin: 1rem 0;
+  border: none;
+  cursor: pointer;
+}
+
+.see-work-btn:hover {
+  background-color: var(--color-white);
+  color: var(--color-secondary);
+}
+
+.see-work-btn:focus {
+  background-color: var(--color-secondary);
+  color: var(--color-white);
+}
+
+.see-work-btn:disabled {
+  background-color: var(--color-white);
+  color: #5b5957;
+}
```

Comment:
- [ ] Great job adding the hover effect. Kindly make your hover and active state match the project template. You are expected to stick to the template as much as possible.
![image](https://user-images.githubusercontent.com/94551484/235269973-3197e00f-22f2-442f-9434-0963e1a175de.png)

Reply:
@Kamba56 I didnt understand your comment fully, can you please explain more in details